aiida/backends/djsite/db/migrations/0046_inline_calc_attributes.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Migrate attributes from old InlineCalculation that have been transformed into CalcFunctions:
+
+* OLD `sealed` and `function_name` remain the same
+* OLD `namespace` becomes `function_namespace`
+* OLD `first_line_source_code` becomes `function_starting_line_number`
+* OLD `source_file` is moved to repository and dropped from DB
+* OLD `source_code` is used to calculate `function_number_of_lines` and dropped from DB
+* NEW `exit_status` is initialized at 0
+* NEW `process_state` is initialized at "finished"
+* NEW `process_label` is set as `Legacy InlineCalculation`
+
+Note 1: old InlineCalculation can be identified by the old keywords, and after migration they will be
+identified by having `function_number_of_lines` but no `version`.
```

Comment:
This is not true I think. Tthere was a time when `CalcFunctionNodes` were created directly but without a `version` attribute.

Reply:
Yeah, I meant the combination of both, since other `CalcFunctionNodes` don't have `function_number_of_lines`. They could if, when merging the PR that starts keeping track of this new attribute, a migration for older `CalcFunctionNodes` is included, but I think this is not certain since it would involve findig a way to parse the file to get the number of lines and so (I think we discussed this, no?)

Anyways, I added this as a way to identify them before deciding to set the new `process_label` to `Legacy InlineCalculation` instead of just copying the `function_name` there, so I think now that `process_label` should be enough of an identifier and I can just remove the note.